### PR TITLE
SmartHub: add Markapp alias

### PR DESCRIPTION
**Pull request by SmartHubSolutions**

### CODE REVIEW

File: static/bidder-info/smarthub.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-endpoint: "http://{{.Host}}-prebid.smart-hub.io/?seat={{.AccountID}}&token={{.SourceId}}"
+endpoint: "http://prebid.smart-hub.io/pbserver?partnerName={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"
```
[the remark this answers is not in the document]
Comment (q67) [reply static/bidder-info/smarthub.yaml:1]: we had some problems but from now on the server will be working 24/7

[the remark this answers is not in the document]
Comment (SmartHubSolutions) [reply static/bidder-info/smarthub.yaml:1]: we had some problems but from now on the server will be working 24/7

[the remark this answers is not in the document]
Comment (SmartHubSolutions) [reply static/bidder-info/smarthub.yaml:1]: link must be ...pbserver?partnerName... instead wrong ...pbserver/partnerName...

[the remark this answers is not in the document]
Comment (onkarvhanumante) [reply static/bidder-info/smarthub.yaml:1]: received 200 response
```
curl -i --location --request POST 'http://prebid.smart-hub.io/pbserver?partnerName=partnertest&seat=9Q20EdGxzgWdfPYShScl&token=zpl5iB5Ugpe9ofVTzi44WzfjZZYq1yer'
HTTP/1.1 200 OK
Server: nginx
```

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**